### ISSUE TITLE
CI: invalid command `bdist_wheel`, connection error in nightly static analysis run

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop#4902]
**Example:**
https://circleci.com/gh/freedomofpress/securedrop/35296

**Output excerpt:**
```
  Running setup.py clean for cryptography
  Running setup.py bdist_wheel for flask-assets: started
  Running setup.py bdist_wheel for flask-assets: finished with status 'error'
  Complete output from command /opt/venvs/securedrop-app-code/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-0olg4840/flask-assets/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /tmp/tmpnrs5a21ipip-wheel- --python-tag cp35:
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help
  
  error: invalid command 'bdist_wheel'
  
  ----------------------------------------
  Failed building wheel for flask-assets
  Running setup.py clean for flask-assets
  Running setup.py bdist_wheel for itsdangerous: started
  Running setup.py bdist_wheel for itsdangerous: finished with status 'error'
  Complete output from command /opt/venvs/securedrop-app-code/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-build-0olg4840/itsdangerous/setup.py';exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'), __file__, 'exec'))" bdist_wheel -d /tmp/tmpu97v42gmpip-wheel- --python-tag cp35:
  usage: -c [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: -c --help [cmd1 cmd2 ...]
     or: -c --help-commands
     or: -c cmd --help
  
  error: invalid command 'bdist_wheel'
...
  ```

Connection error occurs later in output:

```
Collecting flask-testing==0.7.1 (from -r requirements/python3/test-requirements.txt (line 17))

(...)

ConnectionResetError: [Errno 104] Connection reset by peer
```

**Comment by emkll** [comment]
I have seen the `bdist_wheel` error locally, i think it's due to a lacking python-wheel package (or wheel pip package)

The connection reset error appears to be related to https://github.com/freedomofpress/securedrop/issues/4898

**Comment by zenmonkeykstop** [comment]
This was resolved by #5001 - closing.
